Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1704.cs

```diff
@@ -0,0 +1,485 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.CustomAttributes;
+
+#if UITEST
+using Xamarin.UITest;
+using Xamarin.Forms.Core.UITests;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 1704, "[Enhancement] Basic GIF animation features", PlatformAffected.UWP)]
+	public class Issue1704 : TestTabbedPage
+	{
+		ContentPage _page1;
+		ContentPage _page2;
+		ContentPage _page3;
+		ContentPage _page4;
+
+		protected override void Init()
+		{
+			_page1 = new OnLoadAnimationPage { Title = "On Load" };
+			_page2 = new OnStartAnimationPage { Title = "On Start" };
+			_page3 = new LoadImageSourceAnimationPage { Title = "Source" };
+			_page4 = new MiscPage { Title = "Misc" };
+
+			Children.Add(_page1);
+			Children.Add(_page2);
+			Children.Add(_page3);
+			Children.Add(_page4);
+		}
+
+#if UITEST
+		[Test]
+		[Category(UITestCategories.ManualReview)]
+		public void Issue1704Test()
+		{
+			RunningApp.WaitForElement("On Load");
```

Comment:
Here can you click on the tabs and buttons just to make sure it doesn't crash?  You can just use the text for the tabs to switch between tabs
```C#
RunningApp.Tap(q => q.Marked("ON START"));
RunningApp.WaitForElement(q => q.Marked("Start Animation"));
RunningApp.Tap(q => q.Marked("Start Animation"));
etc...
```

Reply:
Added.